src/test/resources/features/acceptance/test-nhs-number.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Enter NHS number
+  In order to apply for the Apply for Healthy Start programme
+  As a potential claimant
+  I want to enter my NHS number
+
+  Background:
+    Given I am in nhs number page
```

Comment:
I do think that the existing step reads better, so this would be `I navigate to the nhs number page`.

Reply:
changed

Reply:
It doesn't look like you have pushed any of these changes to this branch for me to re-review.